inscrawler/crawler.py

```diff
@@ -78,17 +78,19 @@ def login(self):
         browser = self.browser
         url = "%s/accounts/login/" % (InsCrawler.URL)
         browser.get(url)
-        u_input = browser.find_one('input[name="username"]')
+        u_input = browser.find_one(
+            'input[aria-label="Phone number, username, or email"]')
         u_input.send_keys(secret.username)
-        p_input = browser.find_one('input[name="password"]')
+        p_input = browser.find_one('input[aria-label="Password"]')
```

Comment:
Why should we change fetching inputs by name to fetching inputs by **aria-label** ?